app/javascript/components/patient/laboratory/Laboratory.js

```diff
@@ -1,29 +1,21 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Button } from 'react-bootstrap';
+import { Button, Dropdown } from 'react-bootstrap';
 import axios from 'axios';
-
 import LaboratoryModal from './LaboratoryModal';
 import reportError from '../../util/ReportError';
+import DeleteDialog from '../../util/DeleteDialog';
 
 class Laboratory extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      showModal: false,
+      showEditModal: false,
```

Comment:
this actually toggles the modal for adding and editing labs... thoughts on calling this `showLabModal` and updating all edit references to use this?